_docs/tasks/security/plugin-ca-cert.md

```diff
@@ -9,11 +9,11 @@ type: markdown
 ---
 {% include home.html %}
 
-This task shows how operators can plug existing root certificate, signing certificate and key into Istio Citadel.
+This task shows how operators can plug existing root certificate, signing certificate and key into Citadel.
```

Comment:
This task shows how operators can configure Citadel with existing root certificates, signing certificates and keys.

Reply:
Done.